spark/src/main/scala/org/apache/spark/sql/delta/PreprocessTableWithDVs.scala

```diff
@@ -145,7 +144,7 @@ object ScanWithDeletionVectors {
       new SerializableConfiguration(tahoeFileIndex.deltaLog.newDeltaHadoopConf()))
 
     val newFileFormat = fileFormat.copyWithDVInfo(
-      tahoeFileIndex.path.toString, filePathToDVBroadcastMap, hadoopConfBroadcast)
+      tahoeFileIndex.path.toString, hadoopConfBroadcast)
```

Comment:
I would remove `copyWithDVInfo` at this point and directly do here:
```
val newFileFormat = fileFormat.copy(isSplittable = false, disablePushDowns = true)
```


Or rename `copyWithDVInfo` to e.g. `disableSplittingAndPushdown`